wicketd/tests/integration_tests/inventory.rs

```diff
@@ -45,5 +49,64 @@ async fn test_inventory() {
     // 4 SPs attached to the inventory.
     assert_eq!(inventory.sps.len(), 4);
 
+    // Test CLI command
+    {
+        let args = vec!["inventory", "configured-bootstrap-sleds", "--json"];
+        let mut stdout = Vec::new();
+        let mut stderr = Vec::new();
+        let output = OutputKind::Captured {
+            log: wicketd_testctx.log().clone(),
+            stdout: &mut stdout,
+            stderr: &mut stderr,
+        };
+
+        wicket::exec_with_args(wicketd_testctx.wicketd_addr, args, output)
+            .await
+            .expect("wicket inventory configured-bootstrap-sleds failed");
+
+        // stdout should contain a JSON object.
+        let response: Vec<BootstrapSledDescription> =
+            serde_json::from_slice(&stdout).expect("stdout is valid JSON");
+
+        // This is the data we have today but I want it to be different from
+        // this to test having an address and such
```

Comment:
Is this comment left over from an earlier iteration?

Reply:
oh yeah sure is